### ISSUE TITLE
Recent Contacts/ Favourites

### ISSUE THREAD

**Issue by monikasmolarek** [spacemeshos/smapp#1240]
### Rationale

On the _Contacts_ screen, we have a "Recent" section that is not implemented. 
It would be good to do something about it - either implement or remove it. 

If possible and not too complicated, I would suggest maybe implementing something like a "Favourites" contacts. So that the Smappers could mark themselves the contacts as favorites and then be able to find them easily on the list by using a filter or sorting them out on top of the list. 
 
<img width="1312" alt="Screenshot 2023-04-27 at 20 34 15" src="https://user-images.githubusercontent.com/89876259/234968871-637bbfc8-2e8d-4158-bb71-aa33c1e3c08f.png">

### Prerequisites
/ 
### User story

If "Recent" not removed, then: 

- [ ] I open the CONTACTS screen
- [ ] I can see the list of several recently used contacts
- [ ] I can use those contacts to trigger the "Send" transaction

### Acceptance criteria

- [ ] The basic scenario works as described
- [ ] The CONTACTS screen is well-designed (in terms of UX)
- [ ] The list shows the recently used contacts correctly 
- [ ] The list changes dynamically after we use some other contacts
- [ ] The list correctly reflects the Contacts changes (editing, deletion)
- [ ] This section should not contain too many records - 3 or 5 would be enough?

### How to implement

Maybe the "Create new contact" button would need some changes - size/ position/ shape?

### How to test

check the Acceptance criteria

**Comment by brusherru** [comment]
I think that the contacts are already about your "favorites".
It is not a phone or messenger to distinguish them in a such fashion.

Moreover, some of the possible favorites are useless to add to the contacts list because of privacy/security/other reasons. For example, let's say that you have an account on some CEX (like Binance), and you're depositing some money there from time to time. And it seems that the address that you're using might be added there. BUT the reality is so, that you'd better open Binance and check the address once again because they might change it for some reason (it happens sometimes), and if you send money to the wrong address — nobody can do anything with it (probably except Binance in this example) :)

So most likely you'll add there some of your friends or multi-sig account, in which you have a part.

Regarding recent contacts — I believe it is useless on this screen, so we can get rid of easily :)
In the rest, it might be useful on the "Send transaction" screen to have "Recent" in the dropdown (just to have more easy access to contacts that you're interacting with more often).
And it is doable (we have contacts and transaction log -> we can sort out recent contacts without storing any additional data). But definitely is not a high priority stuff :)

To sum up:
1. I propose just removing "Recent" from the contacts page
2. Do not do "favorites"
3. Add recent to the contacts dropdown in the future, after Genesis, if we will find it useful :)

**Comment by monikasmolarek** [comment]
I agree @brusherru. I will remove this "recent" section and close this issue, as I believe this topic should not remain here in any kind of backlog for months. If someone needs the recently used addresses then he could go to the transaction log and check the last few records.
